modulefiles/OznMonBuild.jet

```diff
@@ -0,0 +1,17 @@
+#%Module################################################
+# Modfule file for oznmon
+########################################################
+export ver=2.0.2
+export FCOMP=mpiifort
+
+export CF=$FCOMP
```

Comment:
What is `CF`?

Reply:
Please check with the original developer. I use the same approach:
grep "export CF" *
OznMonBuild.jet:export CF=$FCOMP
OznMonBuild.orion:export CF=$FCOMP
OznMonBuild.wcoss_dell_p3:export CF=$FCOMP
RadMonBuild.jet:export CF=$FCOMP
RadMonBuild.orion:export CF=$FCOMP
RadMonBuild.wcoss_dell_p3:export CF=$FCOMP

Reply:
Yeah, I don't think that is correct.
`CF` is not used in the GSI.  I checked.
These flags being set here are also ignored. 

I am sure you can eliminate lines 5-12 from all these files and it will be identical. 

Also, if the PR is adding functionality provided by some other developers, please invite them in the review.

Reply:
Same comment as above, please remove the modulefiles/OznMonBuild.* and modulefiles/RadMonBuild.* modulefiles. Thanks!

Reply:
Removed as requested.